src/main/resources/application.properties

```diff
@@ -17,9 +17,11 @@ AWS_ROLE_ARN=if_you_see_this_please_use_AWS_Setup
 AWS_WEB_IDENTITY_TOKEN_FILE=if_you_see_this_please_use_AWS_Setup
 FILENAME_CHALLENGE9=wrongsecret
 FILENAME_CHALLENGE10=wrongsecret-2
-azure.keyvault.enabled=false
-management.health.azure-key-vault.enabled=false
-azure.keyvault.uri=https://default.vault.localhost
+spring.cloud.azure.keyvault.secret.credential.client-id=false
+spring.cloud.azure.credential.managed-identity-enabled=false
```

Comment:
Examples showing how to configure credentials can be found here https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#authenticate-with-azure-active-directory.

Reply:
I hoped we could leverage our existing setup (see Lb.yml and the other K8s files where the identity was bound to the pod), but I am just not sure from the docs how to set that up and if that is reusable..

Reply:
Thanks to your latest comment, it now works! Thank you!